server/src/meguca/parser/commands_test.go

```diff
@@ -9,10 +9,12 @@ import (
 	"testing"
 )
 
+var isSlut bool
```

Comment:
Please put this inside the function. No reason to pollute the global scope.

Reply:
aaaaaaaaaaaaaa i hate repetition but okay